README.md

```diff
@@ -2,22 +2,28 @@
 ## By: Derek Tan
 
 ### Summary:
-This repository contains code for a homemade recursive JSON parser in C. Because this is a _toy_ parser, not all JSON standard features are supported. However, this is also a work in progress, so the notes below may not be 100% true.
+This repository contains code for a homemade recursive JSON parser in C. Because this is a toy parser, not all standard JSON features are meant to be supported. I do not intend this to be a production-ready, blazingly fast parser, but I made this for fun. Feel free to fork my work in progress project.
 
 ### Usage:
  - Build: `make all`
- - Run: `./myjson <json file name>`
+ - Run: `./myjson <test number>`
+    - Test 1: Access Array in an Object.
+    - Test 2: Access the first item in a plain Array.
+    - Test 3: Access a property of the second Object in a list of Objects.
  - Clean: `make clean`
 
 ### Caveats:
  1. No backslash escaped characters.
  2. No Unicode support.
- 3. No booleans yet.
- 4. The JSON source is copied into a memory buffer which is inefficient use of memory for larger files.
+ 3. No booleans yet. (I should add this!)
```

Comment:
This statement would be better placed in the **To Do** section during the next batch of changes.